website/src/_app.tsx

```diff
@@ -183,6 +186,9 @@ function MyAppWithLocale({ Component, router, pageProps }: AppProps) {
           </Layout>
         </AnalyticsProvider>
       </GDSProvider>
+      <AskCookbook
+        apiKey="eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiI2NjZiOWUzY2Q3OTg1YTcyNDFlMzQxMjEiLCJpYXQiOjE3MTgzMjg4OTIsImV4cCI6MjAzMzkwNDg5Mn0.RWS5shd1qQq6US4C32qDV8xv5E8TBBTB6gCqoxmHiSw"
```

Comment:
I guess these would be restricted by some CORS to just graph docs? or should we somehow have these injected as secrets during build time/for deployments?

Reply:
Yeah, it's going to be restricted by CORS on our end.

It's also more of a project ID/public key by its nature, and it's going to be exposed in HTTP requests to our backend either way, so it should be fine to just hardcode it here. But let me know if you'd like me to obtain that from an environment variable. Happy to do it :)